packages/suite-data/files/translations/cs.json

```diff
@@ -1166,7 +1166,7 @@
   "TR_SELL_TERMS_4": "Chápu, že kryptoměny jsou nově vznikajícím finančním nástrojem a že v některých oblastech mohou být regulace omezené. To mě může vystavit vyššímu riziku podvodu, krádeže nebo nestability trhu.",
   "TR_SELL_TERMS_5": "Chápu, že transakce s kryptoměnami jsou nevratné a že za svůj nákup nebudu moci dostat zpět peníze.",
   "TR_SELL_TRANS_ID": "Trans. ID:",
-  "TR_SELL_VALIDATION_ERROR_EMPTY": "Požadováno",
+  "TR_REQUIRED_FIELD": "Požadováno",
```

Comment:
just a note and followup TODO: 
changes in translations json files are not persistent, you have to apply those changes in Crowdin but only after the change from messages.ts  is uploaded there.

Reply:
Oops, I was replacing globally, my bad. Will remove

Reply:
Done. Just so that we don't have unnecessary changes.